### PR TITLE
Added further instructions for using the blueprints

### PR DESCRIPTION
**Pull request by Graeme-Miller**

### CODE REVIEW

**Comment by grkvlt** [comment]
I updated `clocker.bom` to remove references to the `main` folder, should be OK now.

File: index.md [hunk 1]
```diff
@@ -13,16 +13,31 @@ Clocker for [Apache Brooklyn](https://brooklyn.apache.org/) is a set of open sou
...
-2. Once this is done, [download the `clocker.bom` file](clocker.bom). It contains all entities required to run containerised solutions. You need to add this file to your [Brooklyn catalog](http://brooklyn.apache.org/v/latest/ops/catalog/index.html). You can achieve this by either drag-and-dropping the file into the YAML editor (composer tab of the Brooklyn UI) or by using the [Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html) as follows:
+To use these blueprints you have two options. You can build the open source Apache Brooklyn and add the BOM files manually. Alternatively you can use a pre-packed application called AMP.
+
+### Apache Brooklyn
+
+1. Firstly, you need to have a build of Apache Brooklyn, if you have not already done this, please [follow these instructions](https://brooklyn.apache.org/v/latest/start/running.html) to install Brooklyn.
+2. You will need to 'mvn clean install' [brooklyn-etcd](https://github.com/brooklyncentral/brooklyn-etcd/) and [Clocker](https://github.com/brooklyncentral/clocker)
+3. Copy the brooklyn-etcd, common, swarm and kubernetes jar into lib/dropins in your Brooklyn directory
+4. Launch Brooklyn
```
Comment (tbouron) [inline index.md]: I would add a link to the doc like so:
```
Launch Brooklyn. To do so, please [follow these instructions](https://brooklyn.apache.org/v/latest/start/running.html).
```

File: index.md [hunk 2]
```diff
@@ -13,16 +13,31 @@ Clocker for [Apache Brooklyn](https://brooklyn.apache.org/) is a set of open sou
...
+6. You will then need to [setup a location](https://brooklyn.apache.org/v/latest/ops/locations/index.html) to which you will deploy these entities. We came up with location's templates to add to your catalog, that you can use out of the box for [AWS](tutorial/locations/aws-example-location.bom), [SoftLayer]((tutorial/locations/sl-example-location.bom)), [Azure](tutorial/locations/azure-example-location.bom), [GCE](tutorial/locations/gce-example-location.bom) and [Blue Box](tutorial/locations/bb-example-location.bom). You can find more information about what is going on in those files [here](tutorial/swarm-cluster.html#setup-a-cloud-location).
+7. The last thing to do is to deploy one (or more) of these new entities. They should be available as an application in the "Create Application" dialog. Check out [our tutorials](tutorials) or you use our [yaml examples](examples/swarm.yaml){:target="blank"} to get started.
```
Comment (tbouron) [inline index.md]: The sentence `Check out [our tutorials](tutorials) or you use our [yaml examples](examples/swarm.yaml){:target="blank"} to get started.` is common to Brooklyn and AMP. It should be displayed below both sections

**Comment by tbouron** [comment]
Another thing @Graeme-Miller. I made the docs for Clocker 2.0 Brooklyn focused, not AMP. This means that in tutorials, I talk exclusively about Brooklyn UI. You might want to change that.

File: index.md [hunk 3]
```diff
@@ -13,16 +13,31 @@ Clocker for [Apache Brooklyn](https://brooklyn.apache.org/) is a set of open sou
...
-2. Once this is done, [download the `clocker.bom` file](clocker.bom). It contains all entities required to run containerised solutions. You need to add this file to your [Brooklyn catalog](http://brooklyn.apache.org/v/latest/ops/catalog/index.html). You can achieve this by either drag-and-dropping the file into the YAML editor (composer tab of the Brooklyn UI) or by using the [Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html) as follows:
+To use these blueprints you have two options. You can build the open source Apache Brooklyn and add the BOM files manually. Alternatively you can use a pre-packed application called AMP.
+
+### Apache Brooklyn
+
+1. Firstly, you need to have a build of Apache Brooklyn, if you have not already done this, please [follow these instructions](https://brooklyn.apache.org/v/latest/start/running.html) to install Brooklyn.
+2. You will need to 'mvn clean install' [brooklyn-etcd](https://github.com/brooklyncentral/brooklyn-etcd/) and [Clocker](https://github.com/brooklyncentral/clocker)
+3. Copy the brooklyn-etcd, common, swarm and kubernetes jar into lib/dropins in your Brooklyn directory
```
Comment (Graeme-Miller) [inline index.md]: not sure I follow this comment, I reference lib/dropins already

**Pull request merged**